### ISSUE TITLE
Feature request: return a balance

### ISSUE THREAD

**Issue by westonal** [chubin/rate.sx#2]
Example:

```
curl rate.sx/1Nh7uHdvY6fNwtQtM1G5EZAFPLC33B59rB
```

**Comment by chubin** [comment]
Yes, I accept your idea to access state/balance of some ID or TXID.

But my idea was that you could specify the id direct in some expression (I don't know if we need it, but maybe it could be useful). Something like 1BTC+BTC:1Nh7....rB something like that, so you could directly manipulate with balances, convert them, sum together and so on.

**Comment by westonal** [comment]
Yeah, that sounds great.

**Comment by chubin** [comment]
On the other hand, we need something to chose the currency in the ID/TXID display.
I'm still not sure. Let us try to do something and we will see if it looks good or not.

But I'm still of the opinion that it would be cool to have something to specify a chain
directly in an expression. May be just adding dot `.` or `1.` before? Or `^`?
Whatever. Some prefix would be great.

**Comment by westonal** [comment]
How do you mean? like this:
```
curl rate.sx/btc.1Nh7uHdvY6fNwtQtM1G5EZAFPLC33B59rB
474.27659636
```
For fiat value:
```
curl usd.rate.sx/btc.1Nh7uHdvY6fNwtQtM1G5EZAFPLC33B59rB
5,581,744.66
```

Alternative chain:
```
curl rate.sx/bch.1Nh7uHdvY6fNwtQtM1G5EZAFPLC33B59rB
27.00474466
```

For multiple chains, same address:

```
curl rate.sx/btc.bch.1Nh7uHdvY6fNwtQtM1G5EZAFPLC33B59rB
474.27659636 BTC + 27.00474466 BCH
```

```
curl usd.rate.sx/btc.bch.1Nh7uHdvY6fNwtQtM1G5EZAFPLC33B59rB
5,622,440.54
```
For multiple addresses, possibly on different chains:
```
curl usd.rate.sx/btc.bch.1Nh7uHdvY6fNwtQtM1G5EZAFPLC33B59rB+ltc.LTCPodGk5FwcJaRzH9yKnwicxzFUbxp6tN
5,822,440.54
```

So the syntax is:

```
<expression> = <chains>.<address> | <expression> + <expression>
<chains> = <chain> | <chain>.<chains>
```

**Comment by chubin** [comment]
It's cool. I like this Idea with the dot. It looks like a hierarchical notation, as if you specify some domain (btc domain or bch domain; not a DNS domain but just a general domain). From this point of view, it is a little bit wrong to join chains with a dot: btc.bch.XXX. May be btc,bch.XXX? or even b*.XXX if you want to say "I need all b-currencies with this address".

**Comment by westonal** [comment]
I'd say accept one at first. If people want to see multiple, for the meantime they can just:

```
curl usd.rate.sx/btc.1Nh7uHdvY6fNwtQtM1G5EZAFPLC33B59rB+bch.1Nh7uHdvY6fNwtQtM1G5EZAFPLC33B59rB
```
